manifest.yml

```diff
@@ -0,0 +1,17 @@
+applications:
+- name: funding-service-design-audit-dev
+  memory: 64M
+  buildpacks:
+  - https://github.com/cloudfoundry/python-buildpack.git
+  command: gunicorn wsgi:app -c run/gunicorn/devtest.py
+  services:
+  - logit-ssl-drain
+
+- name: funding-service-design-audit-test
+  memory: 64M
+  buildpacks:
+  - https://github.com/cloudfoundry/python-buildpack.git
+  command: gunicorn wsgi:app -c run/gunicorn/devtest.py
+  services:
+  - logit-ssl-drain
+  - funding-service-design-audit-db
```

Comment:
I think we only want the real database on test, else it will fail if people add migrations in pull requests. I think this will mean we might need to use a custom workflow instead of the shared one, as we will need separate manifests for dev and test.

Reply:
Okay so I might well be wrong but… I thought this line: [here](https://github.com/communitiesuk/funding-design-service-workflows/blob/main/.github/workflows/deploy.yml#:~:text=command%3A%20push%20%24%7B%7Binputs.app_name%7D%7D%2Ddev), means the workflow will only pick up the dev application from the listed services in the default manifest.yml
```
applications:
- name: funding-service-design-audit-dev
```
I can't see a trace for the -test env in the logs
I might well be missing something though